packages/lesswrong/components/common/ContentItemBody.tsx

```diff
@@ -148,12 +148,37 @@ class ContentItemBody extends Component<ContentItemBodyProps,ContentItemBodyStat
   }
   
   
+  /**
+   * Return whether a given node from the DOM is inside this ContentItemBody.
+   * Used for checking the selection-anchor in a mouse event, for inline reacts.
+   */
+  containsNode(node: Node): boolean {
+    return !!this.bodyRef.current?.contains(node);
+  }
+  
+  /**
+   * Return a text stringified version of the contents (by stringifying from the
+   * DOM). This is currently used only for warning that an inline-react
+   * identifier is ambiguous; this isn't going to be nicely formatted and
+   * shouldn't be presented to the user (for that, go to the source docuemnt and
```

Comment:
it'd be helpful if this was a little more explicit about how to get the source document.